package.json

```diff
@@ -36,7 +36,7 @@
     "eslint-plugin-prettier": "^3.1.2",
     "prettier": "^2.3.2",
     "ts-node-dev": "^1.0.0-pre.44",
-    "typescript": "^3.8.2"
+    "typescript": "^4.5.4"
```

Comment:
There are a bunch of new type errors with this TypeScript version. I'm guessing we'd either need to update type dependencies or fix our TS config.